syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -1019,20 +1030,31 @@ def encrypt(self, protocol="mpc", **kwargs):
             NotImplementedError: If protocols other than the ones mentioned above are queried
 
         """
-        if protocol.lower() == "mpc":
+        if protocol.lower() in ("mpc", "snn", "fss"):
```

Comment:
NITPICK: I know it should not matter in computational speed, but could we use ```{...}``` for ```set```

Reply:
MAYBE: Keep another variable, ```protocol_lower``` and do only once ```protocol_lower = protocol.lower()```